modules/collector-source/pkg/metric/metrics.go

```diff
@@ -23,6 +23,10 @@ const (
 	ServiceSelectorLabels                                 = "service_selector_labels"
 	StatefulSetMatchLabels                                = "statefulSet_match_labels"
 	KubeReplicasetOwner                                   = "kube_replicaset_owner"
+	KubeResourceQuotaSpecResourceRequests                 = "kube_resource_quota_spec_resource_requests"
+	KubeResourceQuotaSpecResourceLimits                   = "kube_resource_quota_spec_resource_limits"
+	KubeResourceQuotaStatusUsedResourceRequests           = "kube_resource_quota_status_used_resource_requests"
+	KubeResourceQuotaStatusUsedResourceLimits             = "kube_resource_quota_status_used_resource_limits"
```

Comment:
I don't think we need to follow this (correct @Sean-Holcomb ?) but here is what the KSM metrics look like: https://github.com/kubernetes/kube-state-metrics/blob/main/docs/metrics/policy/resourcequota-metrics.md

Just interesting because we're _sort of_ following their naming conventions, but we're breaking with some things too.

Reply:
are you suggesting we exclude the `kube_` prefix. I think that might be a good idea to prevent confusion for users who are digging around in the metrics.